config/foreman.migrations/20190111180118_delete_removed_settings.rb

```diff
@@ -1,74 +1,10 @@
-DELETED = {
```

Comment:
I can be blamed for this. Kafo already removed unknown keys and this was never needed in the first place.